extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -650,6 +650,8 @@ protected function setResponseValuesFromConfig( Wikia\Search\Config $searchConfi
 
 	protected function addRightRailModules( Wikia\Search\Config $searchConfig ) {
 		global $wgLang, $wgEnableFandomStoriesOnSearchResultPage;
+		global $wgDisableFandomStoriesSearchResultsCaching;
+		global $wgEnableFandomStoriesSearchLogging;
```

Comment:
Nitpick, I suggest to keep globals consistently formatted ( either single global keyword and all vars separated by commas, or every single one separately; the convention leans towards the former )